src/App.css

```diff
@@ -111,7 +125,63 @@ p {
   bottom: 1rem;
 }
 
-@media only screen and (min-device-width: 360px) and (max-device-width: 812px) {
+.toggle-todo {
+  background-color: transparent;
+  color: rgb(201, 201, 201);
```

Comment:
use css var names

Reply:
I'm not repeating it anywhere. So, I did not use variables for the colors. But I have used color variables for repeating colors.